auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/UmsClient.java

```diff
@@ -812,7 +812,7 @@ public GetRightsResponse getRightsForUser(String requestId, String actorCrn, Str
         if (resourceCrn == null) {
             resourceCrn = "*";
         }
-        return newStub(requestId).getRights(
+        return newShortTimeoutStub(requestId).getRights(
```

Comment:
This is another stub, the checkRight related can be found in AuthorizationClient